compiler/src/codegen/compcore.re

```diff
@@ -31,6 +31,7 @@ let gensym_label = s => {
   gensym_counter := gensym_counter^ + 1;
   Printf.sprintf("%s.%d", s, gensym_counter^);
 };
+let reset_labels = () => gensym_counter := 0;
```

Comment:
What's this about?

Reply:
This resets the label counter between runs (since before if you used the compiler as a library like we do in the tests, it just keeps incrementing) so it makes the snapshots more stable since the numbers show up in the snapshot. (If you ran test files in a different order or maybe filtered, you could maybe get different compiler output, but this takes care of that.)